docs/application/web/api/5.0/device_api/wearable/tizen/ppm.html

```diff
@@ -108,9 +125,8 @@ <h2>1. Type Definitions</h2>
 <div class="brief">
  Specifies the result of a permission request.
           </div>
-<pre class="webidl prettyprint">  enum PermissionRequestResult { "PPM_ALLOW_FOREVER", "PPM_DENY_FOREVER", "PPM_DENY_ONCE" };</pre>
-<p><span class="version">
-            Since: </span>
+<pre class="webidl prettyprint">enum PermissionRequestResult { "PPM_ALLOW_FOREVER", "PPM_DENY_FOREVER", "PPM_DENY_ONCE" };</pre>
```

Comment:
Line 101: </a><h3>1.1. PermissionType</h3>
Line 124: <h3>1.2. PermissionRequestResult</h3>
Line 150: <h3>2.1. PrivacyPrivilegeManagerObject</h3>
Line 168: </a><h3>2.2. PrivacyPrivilegeManager</h3>

Please get <h3>  to next line.

Reply:
This is only body of html file which is displayed in browser. In browser this file is displayed properly. This html file was generated automatically from source file and is consistent with other modules. It shouldn't be changed by hand